pytest_kubernetes/plugin.py

```diff
@@ -52,17 +54,21 @@ def remaining_clusters_teardown():
 
 
 def pytest_addoption(parser):
-    group = parser.getgroup("k8s")
-    group.addoption(
+    k8s_group = parser.getgroup("k8s")
+    k8s_group.addoption(
         "--k8s-cluster-name",
         default="pytest",
         help="Name of the Kubernetes cluster (default 'pytest').",
     )
-    group.addoption(
+    k8s_group.addoption(
         "--k8s-provider",
         help="The default cluster provider; selects k3d, kind, minikube depending on what is available",
     )
-    group.addoption(
+    k8s_group.addoption(
         "--k8s-version",
         help="The default cluster provider; selects k3d, kind, minikube depending on what is available",
     )
+    k8s_group.addoption(
+        "--k8s-cluster-config",
```

Comment:
Rename to `--k8s-cluster-config` provider config?

Reply:
Will also go with `--k8s-provider-config`